src/main/java/cz/cvut/kbss/termit/persistence/dao/TermDao.java

```diff
@@ -315,6 +315,34 @@ public boolean isEmpty(Vocabulary vocabulary) {
         }
     }
 
+    private String orderSentence(String lang, String var) {
```

Comment:
I'd prefer some explanatory comment. Especially given the complex and weird looking "cs" case.